cmd/drone-server/drone.go

```diff
@@ -185,6 +185,14 @@ func main() {
 			repo.GET("/logs/:number/:task", server.GetLogs)
 			// repo.POST("/status/:number", server.PostBuildStatus)
 		}
+
+		// Routes for external services
+		repoExternal := repos.Group("")
+		{
+			repoExternal.Use(server.SetRepo())
+
+			repoExternal.GET("/pr/:number", server.GetPullRequest)
```

Comment:
But this need to access from CI without user session, the same mechanism used in hook handler